.github/workflows/bare.yaml

```diff
@@ -38,16 +39,25 @@ jobs:
       uses: actions/checkout@v2
       # Required to get the correct git rev to use in Python's version.
       with:
-        fetch-depth: 2
+        ref: ${{ github.event.after }}
```

Comment:
Minor comment
can we remove the usage of `actions/checkout@v2` and use `Clone sources independently` everywhere?

Reply:
In the end, I did go for cloning sources independently everywhere, including macOS and Windows. For a lower mental load associated with all this… Thanks!